### DISCUSSION TITLE
Date being passed as Restriction from Main report to subreport

### DISCUSSION THREAD

**Discussion by sridharkannan77** [ariacom/Seal-Report#105] [Bug Reports]
I have a data set where the date range called "Sales date" is the restriction in the main report. This restriction uses the operator "is between" and hence there are 2 values to this restriction

So once the user executes this report, he has to select a date range and the main report should show the count of leads in the main report against a sales person

On clicking the sales person, there is sub report called which will display all the leads against that sales person.
For example, if the date range is "01-Jun-2024" to "12-06-2024" and there are 5 leads on different dates between the mentioned date range against a Sales person A, then the main report should show the count as 5

However since I am calling a sub report on the name, the main report is now also grouping by "Date" which means instead of showing one record against the Salesperson A, it is showing 5 records in the main report itself

I have tried to remove the date from the group by clause.. However there is no way to remove it from the SELECT list

is there any way I can take care of this issue? Any help will be appreciated?

**Comment by ariacom** [comment]
Is that related to the previous discussion ?
Can you provide a sample based on Northwind to check the bug ?
